lib/Router.js

```diff
@@ -250,6 +250,20 @@ Router.prototype.handle = function(request, connection) {
         return request.error(404, "No handlers available for resource " + resourceId);
     }
 
+    if (!!~[Request.METHOD_PATCH, Request.METHOD_POST, Request.METHOD_PUT].indexOf(method)) {
+        var validate = jsen(_.extend({additionalProperties: false}, route.schema));
```

Comment:
If the schema requires a property age and the user doesn't send in a property age for patch ... will this throw an invalid schema?